.github/workflows/end-to-end-tests.yml

```diff
@@ -0,0 +1,36 @@
+name: "end-to-end_tests"
+on:
+    push:
+        branches: [develop]
+    pull_request:
+        branches: [develop]
+jobs:
+    "end-to-end_tests":
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                shell: bash
+                working-directory: ${{ github.workspace }}
+        container:
+            image: vectorim/element-web-ci-e2etests-env:latest
+            env:
+                CI_PACKAGE: true
+                GITHUB_HEAD_REF: ${{ github.head_ref }}
```

Comment:
I guess this is passing along the refs we would need for matching branches, assuming we separate update scripts to use these GitHub vars?